asciidoc-hs.cabal

```diff
@@ -24,12 +24,12 @@ library
                        Text.AsciiDoc.SourceRange
                        Text.AsciiDoc.Token
   -- other-modules:
-  build-depends:       base ^>= 4.14.0.0,
-                       optics-core == 0.3.*,
-                       pandoc-types == 1.21.*,
-                       parsec == 3.1.*,
-                       parser-combinators == 1.2.*,
-                       text == 1.2.*
+  build-depends:       base >= 4.12.0.0,
+                       optics-core >= 0.2,
+                       pandoc-types >= 1.20,
```

Comment:
Agree with the other bound lowering, but in the case of Pandoc I'd probably stick to the latest stable version.

Reply:
I retract, at time being, easy installation is probably more important than cutting edge Pandoc.